src/gimelstudio/core/node/property.py

```diff
@@ -29,13 +29,14 @@ class Property(object):
     """
     The base node Property class.
     """
-    def __init__(self, idname, default, label, visible=True):
+    def __init__(self, idname, default, label, visible=True, mute=True):
```

Comment:
Shouldn't the default value for ``mute`` be ``False``?

Reply:
Yep, that is typo. Had to refork project couse I made a few stupid commits :/

Reply:
> Had to refork project couse I made a few stupid commits :/

I understand. :slightly_smiling_face: